src/status_im/contexts/profile/settings/view.cljs

```diff
@@ -9,18 +9,24 @@
             [status-im.contexts.profile.settings.list-items :as settings.items]
             [status-im.contexts.profile.settings.style :as style]
             [status-im.contexts.profile.utils :as profile.utils]
+            [status-im.feature-flags :as ff]
             [utils.debounce :as debounce]
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
+(defn show-settings-item?
```

Comment:
The way things are named is confusing to me. The function `show-settings-item?` refers to an individual item (truly one) of the Settings screen, but right next we have the function `settings-item-view`, which actually refers to a group of one or more items.

One suggestion I have is to name the group of items as `section` or `group` and the individual item as `item`.

Reply:
yeah I agree, let me try to make things a bit clearer 👍